go/vt/vtgate/engine/session_primitive.go

```diff
@@ -76,7 +75,7 @@ func (s *SessionPrimitive) TryStreamExecute(ctx context.Context, vcursor VCursor
 
 // GetFields implements the Primitive interface
 func (s *SessionPrimitive) GetFields(ctx context.Context, vcursor VCursor, bindVars map[string]*querypb.BindVariable) (*sqltypes.Result, error) {
-	return nil, vterrors.New(vtrpcpb.Code_INTERNAL, "not supported for this primitive")
+	return nil, vterrors.VT13001("not supported by the session primitive")
```

Comment:
```suggestion
	return nil, vterrors.VT13001("not supported by SessionPrimitive")
```

Reply:
resolved in [f2351bb](https://github.com/vitessio/vitess/pull/10738/commits/f2351bba65f347a8e5ce72ff87757ccaf20115af)